tests/pipelines/main.from_assemblies.nf.test

```diff
@@ -251,9 +253,12 @@ nextflow_pipeline {
                 skip_checkm = true
                 skip_raw_read_metrics = true
                 skip_polishing = true
-                
+
                 skip_staramr = false
                 skip_subtyping = false
+                skip_allele_calling = true
+
+                params.allele_scheme = "$baseDir/tests/databases/locidex_salm"
```

Comment:
This allele scheme seems to be added to the `listeria` set of tests, but it's for salmonella. Could you add it to the salmonella genome test case instead? Also, it is unused correct since you have `skip_allele_calling = true`?

For testing, could I have you:

1. Add a line like this https://github.com/phac-nml/mikrokondo/blob/c0467b7bcf8eeba7a6e83113324a5636715b7fd4/tests/pipelines/main.from_assemblies.nf.test#L89, but for salmonella to check that the profiles file is listed in the `iridanext.output.json.gz` file?
2. Store the example output profiles file for this genome someplace in the test data (assuming it's not too big).
3. We'll do a comparison to make sure the mlst profiles file we produce matches the expected file stored in the test data.

Reply:
fixed and updated in: 30cb619c42186c5aa840e4064aab6994715c7440